src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int FIGURE_COUNT = 6;
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[FIGURE_COUNT];
+        for (int i = 0; i < FIGURE_COUNT - 1; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].draw();
```

Comment:
not implemented: `The first half of figures in this array should be generated with random parameters.`